general/src/pii/builtin.rs

```diff
@@ -49,11 +49,12 @@ declare_builtin_rules! {
             rules: vec![
                 "@ip".into(),
                 "@email".into(),
-                "@creditcard".into(),
+                "@creditcard:replace".into(),
```

Comment:
If `@creditcard:replace` is the rule referenced by `@common`, can we make it the `@creditcard` alias, instead of what is set right now?